Makefile

```diff
@@ -232,3 +228,10 @@ test-e2e:
 .PHONY: test-doc-automation
 test-doc-automation:
 	$(RUN_GINKGO) $(GINKGO_FLAGS_ONE) --junit-report="test-doc-automation.xml"  tests/documentation/...
+
+
+# Generate OpenAPISpec library based on ododevapispec.yaml inside pkg/apiserver-gen; this will only generate interfaces
+# Actual implementation must be done inside pkg/apiserver-impl
+.PHONY: generate-apiserver
+generate-apiserver: ## Generate OpenAPISpec library based on ododevapispec.yaml inside pkg/apiserver-gen; also go fmt the files
+	openapi-generator generate -i ododevapispec.yaml -g go-server -o pkg/apiserver-gen --additional-properties=outputAsLibrary=true,onlyInterfaces=true && echo "Formatting generated files:" && go fmt ./pkg/apiserver-gen/...
```

Comment:
As discussed the other day, can we explore the possibility to run the generator CLI from a container? I can see one additional benefit in doing so: enforcing a specific version of the generator CLI, to get consistent output.

Reply:
My bad; I forgot to change this one. I'll fix it.